src/Engine/Component/RigidBody.hpp

```diff
@@ -81,13 +94,20 @@ namespace Component {
             // nullptr is returned.
             btRigidBody* GetBulletRigidBody();
 
+            // Get the underlying Bullet collision object. If none has been
+            // set, nullptr is returned. If the rigid body is not a ghost,
+            // the returned collision object is the rigid body.
+            btCollisionObject* GetBulletCollisionObject();
```

Comment:
No @return in the doxygen comments

Reply:
It's a private member. We don't do doxygen for those :)